app/models/teammate.rb

```diff
@@ -0,0 +1,97 @@
+class Teammate < ActiveRecord::Base
+
+  PENDING = "pending"
+  ACCEPTED = "accepted"
+  DECLINED = "declined"
+  STATES = ["pending", "accepted", "declined"]
+
+  STAFF_ROLES = ['reviewer', 'program team', 'organizer']
+
+  belongs_to :event
+  belongs_to :user
+
+  validates_uniqueness_of :email, scope: :event
+  validates :email, :event, :role, presence: true
+  validates_format_of :email, :with => /@/
+
+  scope :for_event, -> (event) { where(event: event) }
+  scope :recent, -> { order("created_at DESC") }
```

Comment:
when I usually see recent I assume it does two things, orders by some date and limits the count to some number/scope that represents new content only.

I would look up where this is used and rename to something like 'newest_first' or 'by_created_at'.